services/blockchain-connector/methods/tests/decoder.js

```diff
@@ -16,8 +16,9 @@
 const {
 	decodeBlock,
 	decodeTransaction,
-	decodeEventPayload,
+	decodeSubscriptionEventPayload,
```

Comment:
Please rename `decodeSubscriptionEventPayload` to `decodeAPIClientEventPayload` everywhere.
```suggestion
	decodeAPIClientEventPayload,
```

Reply:
applied.